src/shared/components/palette/base-field/field-read.component.spec.ts

```diff
@@ -63,13 +73,60 @@ describe('FieldReadComponent', () => {
 
   let formGroup: FormGroup = new FormGroup({});
   let caseFields: CaseField[] = [CASE_FIELD];
+  let caseEditComponentStub: any;
```

Comment:
@Daniel-Lam - We had to add CaseEditComponent reference to FieldReadComponent, so that it can be accessed by the child component(s). These lines of code were to declare and add reference to CaseEditComponent in the spec file.